lib/api-pagination.rb

```diff
@@ -87,14 +87,21 @@ def pagy_pages_from(pagy)
       end
     end
 
-    def paginate_with_kaminari(collection, options, paginate_array_options = {})
+    def paginate_with_kaminari(collection, options)
       if Kaminari.config.max_per_page && options[:per_page] > Kaminari.config.max_per_page
         options[:per_page] = Kaminari.config.max_per_page
       elsif options[:per_page] <= 0
         options[:per_page] = get_default_per_page_for_kaminari(collection)
       end
 
       if collection.is_a?(Array) || (options.key?(:paginate_array) && !!options[:paginate_array])
+        # These options are needed for kaminari to paginate an array
+        # https://github.com/kaminari/kaminari/blob/master/kaminari-core/lib/kaminari/models/array_extension.rb#L70
```

Comment:
Can you replace this with the permalink to the exact commit sha (not master)? See discussion here for the Github keyboard shortcut to get this: https://adromance.slack.com/archives/C0CP8GPPT/p1585254543044900

Reply:
Thanks good to know